commons/src/main/kotlin/com/simplemobiletools/commons/activities/ManageBlockedNumbersActivity.kt

```diff
@@ -38,7 +38,10 @@ class ManageBlockedNumbersActivity : BaseSimpleActivity(), RefreshRecyclerViewLi
         updatePlaceholderTexts()
 
         block_unknown.apply {
-            setText(if (baseConfig.appId.startsWith("com.simplemobiletools.dialer")) R.string.block_unknown_calls else R.string.block_unknown_messages)
+            setText(if (baseConfig.appId.startsWith("com.simplemobiletools.dialer"))
+                R.string.block_unknown_calls
+            else
+                R.string.block_unknown_messages)
```

Comment:
omg, just move the if/else out of brackets, this is getting worse

Reply:
ok, moved